server.js

```diff
@@ -97,19 +97,19 @@ app.use(function (req, res, next) {
 app.use(require('./server/coffee'))
 
 var http = require('http')
-app.use('/api', function (req, res) {
-  const opts = { method: req.method }
-
-  if (Math.random() < opbeansRedirectProbability) {
-    const service = opbeansServiceUrls[Math.floor(Math.random() * opbeansServiceUrls.length)]
-    opts.hostname = service.hostname
-    opts.port = service.port
-    opts.path = req.originalUrl
-  } else {
-    opts.hostname = conf.server.hostname
-    opts.port = conf.server.port2
-    opts.path = req.url // req.url will just contain the sub-path without the /api
+app.use('/api', function (req, res, next) {
+   if (Math.random() > opbeansRedirectProbability) {
```

Comment:
```suggestion
  if (Math.random() > opbeansRedirectProbability) {
```

Reply:
This was because of a bug in the GitHub suggestion feature

Reply:
Ha, I thought I could just merge this suggestion my self. Turns out you can't merge your own suggestions 🤦‍♂️

Reply:
Hack: I'll assign my self

Reply:
Or not... I guess it's beta for a reason 😅 

![image](https://user-images.githubusercontent.com/10602/48063648-fe890680-e1c5-11e8-85ad-f8074acac751.png)